packages/compiler-sfc/__tests__/compileScriptPropsTransform.spec.ts

```diff
@@ -99,6 +99,19 @@ describe('sfc props transform', () => {
     assertCode(content)
   })
 
+  test('default values w/ type declaration & key is string', () => {
+    const { content } = compile(`
+      <script setup lang="ts">
+      const { foo = 1 } = defineProps<{ "foo": number }>()
+      </script>
+    `)
+    expect(content).toMatch(`
+  props: {
+    "foo": { type: Number, required: true, default: 1 }
+  },`)
+    assertCode(content)
```

Comment:
Make sure `bindings` here is `{ 'foo': 'props' }` instead of `{ '"foo"': 'props' }`.